mmdet/models/roi_heads/cascade_roi_head.py

```diff
@@ -380,8 +378,9 @@ def simple_test(self, x, proposal_list, img_metas, rescale=False):
                     mask_pred = mask_results['mask_pred']
                     # split batch mask prediction back to each image
                     mask_pred = mask_pred.split(num_mask_rois_per_img, 0)
-                    aug_masks.append(
-                        [m.sigmoid().cpu().numpy() for m in mask_pred])
+                    aug_masks.append([
+                        m.sigmoid().cpu().detach().numpy() for m in mask_pred
```

Comment:
do we still need detach() here?

Reply:
If user forget use `with torch.no_grad()`, detrectly applying `.numpy()` to this tensor would throw the error